### ISSUE TITLE
Should `initialize()` accept `ignoreTranslateChildren`?

### ISSUE THREAD

**Issue by thchia** [ryandrewjohnson/react-localize-redux#89]
**Do you want to request a _feature_ or report a _bug_?**
Possibly bug, but would like to confirm.

**What is the current behavior?**
In v2, you could dispatch the `initialize` action with an option to `ignoreTranslateChildren`. It seems that functionality no longer exists, and I'm wondering if it is intentional or not.

**What is the expected behavior?**
To be able to call the `initialize` prop from `withLocalize` with `ignoreTranslateChildren` as true and block all attempts to `addDefaultTranslation()`.

**Which versions of `react` and `react-localize-redux` are you using?**
`react-localize-redux@v3.0.2`

@ryandrewjohnson if you're good with me re-implementing this functionality, feel free to assign this one back to me and I'll pick it up.

**Comment by ryandrewjohnson** [comment]
In v3.0 the `ignoreTranslateChildren` was purposefully removed from `initialize` options, and instead added as an [option](https://ryandrewjohnson.github.io/react-localize-redux-docs/#translate-2) to `Translate`.

The original thinking is this option was more likely going to be used as an individual override rather than something that someone would want to set globally. Do you foresee a use case where the global setting would be useful?

If so we can add the option back to `initialize` options, and have the `Translate` option act as override for the `initialize` option. Let me know your thoughts.

**Comment by thchia** [comment]
Fair point, it happens that in v2 I was using 'global' flag so for me personally, having the option back would be helpful.

This possibly ties into #85 - I had originally used this setting because of some choking/blocking when rendering a component where `addDefaultTranslation` was called multiple times. Since I switched that off the issue slipped from my mind but perhaps it's a good time to revisit.

Anyway, my vote is to allow this to be set during the initialization call.

**Comment by ryandrewjohnson** [comment]
Ok if you would like to go ahead and create a PR with the addition that's fine with me. In addition to the source updates the [docs](https://github.com/ryandrewjohnson/react-localize-redux-docs) will need to be updated as well.
